### PR TITLE
fix: (CXSPA-979) - Dialogs announcement JAWS fix

### PR DESCRIPTION
**Pull request by Pio-Bar**
Ticket: [CXSPA-979](https://jira.tools.sap/browse/CXSPA-979)

Fixes a JAWS specific bug where the title page was announced while dialog was opened. By providing a label to the dialog container we hear it instead.

### CODE REVIEW

**Review by Zeyber — DISMISSED** [review]

File: feature-libs/cart/saved-cart/components/saved-cart-form-dialog/saved-cart-form-dialog.component.html [hunk 1]
```diff
@@ -4,12 +4,13 @@
     (esc)="close('Escape clicked')"
     class="cx-saved-cart-form-dialog"
     role="dialog"
+    aria-labbeledby="dialogTitle"
```
Comment (developpeurweb) [inline feature-libs/cart/saved-cart/components/saved-cart-form-dialog/saved-cart-form-dialog.component.html]: small typo, it should read `aria-labelledby`

File: feature-libs/quote/components/confirm-dialog/quote-confirm-dialog.component.html [hunk 2]
```diff
@@ -5,14 +5,15 @@
   (esc)="dismissModal('Escape clicked')"
   role="dialog"
   aria-modal="true"
+  aria-labelldby="dialogTitle"
```
Comment (developpeurweb) [inline feature-libs/quote/components/confirm-dialog/quote-confirm-dialog.component.html]: small typo, it should read `aria-labelledby`

**Review by developpeurweb — CHANGES_REQUESTED** [review]
Small adjustments please:
1. Small typos were preventing the dialog from being identified.
2. For some reason, the "Add To Cart" dialog doesn't get identified as a dialog, it seems the focus lands one Tab away from the close button still trapped and narrating the page underneath. Once moving focus to the close button, the dialog gets identified. The ideal would be for the focus to move to the close button when opening, as with the other dialogs.

**Comment by Pio-Bar** [comment]
The 'added to cart' dialog should no longer lose focus after the cart is updated and stay on the close button. Typos fixed, good catch ;)

**Review by rmch91 — APPROVED** [review]

**Review by developpeurweb — APPROVED** [review]

**Pull request merged**